vispy/testing/_runners.py

```diff
@@ -111,6 +111,25 @@ def _unit(mode, extra_arg_string, coverage=False):
         os.rename(op.join(cwd, '.coverage'), out_name)
 
 
+def _docs():
+    """test docstring paramters
+    using vispy/utils/tests/test_docstring_parameters.py"""
+    print("warning: this imports vispy from the vispy installation. it is"
+          " recommended to setup vispy using 'python setup.py develop'"
+          " so that the latest sources are used automatically")
```

Comment:
In that case, I think I'll spend a few hours tonight documenting development for Vispy (the environment (some opengl backend, nosetests, flake), offline testing, CI.